action/protocol/staking/read_state.go

```diff
@@ -123,3 +137,30 @@ func getPageOfBuckets(buckets []*VoteBucket, offset, limit int) []*VoteBucket {
 	}
 	return res
 }
+
+func toIoTexTypesCandidateListV2(candidates CandidateList) *iotextypes.CandidateListV2 {
```

Comment:
how about make this func (l CandidateList) toIotexTypes()?

just like func (d *Candidate) toIotexTypes()

Reply:
I think it is fine. it corresponding to `toIoTexTypesVoteBucketList`